cylc/flow/cfgspec/workflow.py

```diff
@@ -1334,6 +1336,21 @@ def get_script_common_text(this: str, example: Optional[str] = None):
                     "[platforms][<platform name>]submission retry delays"
                 )
             )
+            Conf(
+                'run mode', VDR.V_STRING,
+                options=list(RunMode.OVERRIDING_MODES) + [''],
```

Comment:
Allows it to be unset.

Reply:
We don't want it to be unset do we?

Reply:
I was going to allow it, but I see your point. I was using unset as a proxy for "workflow". I'll remove skip mode from the list allowed by Cylc Play then because it won't do anything useful.